packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -229,9 +234,9 @@ export async function* serveWithVite(
         'externalMetadata'
       ] as ExternalResultMetadata;
       const implicitServerFiltered = implicitServer.filter(
-        (m) => removeNodeJsBuiltinModules(m) && removeAbsoluteUrls(m),
+        (m) => !isNodeJsBuiltinModules(m) && !isAbsoluteUrls(m),
```

Comment:
Node.js has a helper function for this now: https://nodejs.org/api/module.html#moduleisbuiltinmodulename